projects/elife.yaml

```diff
@@ -213,6 +213,13 @@ defaults:
             ec2:
                 # should be the same as basebox '18.04' and 'standalone18.04' aws-alt configurations
                 ami: ami-0b425589c7bb7663d # bionic, build 20180814, hvm:ebs-ssd
+
+        next-salt:
```

Comment:
maybe `standalone-next-salt` since it has `masterless`?

Reply:
weren't we deprecating the long alt-config names because they screwed with something in CI?

`sNextSalt` perhaps?